problemi/boltzmannDistribution/boltzmannDistribution.cpp

```diff
@@ -0,0 +1,144 @@
+#include <TCanvas.h>
+#include <TF1.h>
+#include <TH1D.h>
+#include <TLegend.h>
+
+#include <iomanip>
+#include <iostream>
+#include <sstream>
+#include <string>
+#include <vector>
+
+#include "boltzmann/boltzmann.h"
+
+#define DOCTEST_CONFIG_IMPLEMENT_WITH_MAIN
+#include "doctest.h"
+
+void plotDistributionROOT(const std::vector<int> &interactions_list,
```

Comment:
```suggestion
void plotDistributionROOT(std::vector<int> const& interactions_list,
```
usiamo _east const_ e il `&` attaccato al tipo